src/E1_process_results.py

```diff
@@ -722,3 +727,72 @@ def convert_costs_to_dataframe(dict_values):
     df_pie_plot.iloc[-1, 0] = "Total"
 
     return df_pie_plot
+
+def convert_scalars_to_dataframe(dict_values):
+    """
+    Processes the scalar system-wide KPI so that they can be included in the report
+
+    Parameters
+    ----------
+    dict_values: dict
+        output values of MVS
+
+    Returns
+    -------
+    kpi_scalars_dataframe: :pandas:`pandas.DataFrame<frame>`
+        Dataframe to be displayed as a table in the report
+
+    Notes
+    -----
+    Currently, as the KPI_SCALARS_DICT does not hold any units, the table printed in the report is unit-les.
+    """
+
+    units_cost_kpi = get_units_of_cost_matrix_entries(dict_values[ECONOMIC_DATA], dict_values[KPI][KPI_SCALARS_DICT])
+
+    kpi_scalars_dataframe = pd.DataFrame(dict_values[KPI][KPI_SCALARS_DICT], index=[VALUE])
+    kpi_names = kpi_scalars_dataframe.columns
+    kpi_scalars_dataframe = kpi_scalars_dataframe.transpose()
+    kpi_scalars_dataframe[KPI] = kpi_names
+    kpi_scalars_dataframe[UNIT] = units_cost_kpi
+    kpi_scalars_dataframe = kpi_scalars_dataframe[[KPI, UNIT, VALUE]]
+
+    return kpi_scalars_dataframe
+
+def get_units_of_cost_matrix_entries(dict_economic, kpi_list):
+    """
+    Determines the units of the costs KPI to be stored to :pandas: DataFrame.
+
+    Parameters
+    ----------
+    dict_economic:
+        Economic project data
+
+    KPI_COST_MATRIX_ENTRIES:
```

Comment:
```suggestion
    kpi_list:
```

If `KPI_COST_MATRIX_ENTRIES` is the default, you can add this information below with `Default: KPI_COST_MATRIX_ENTRIES`

Reply:
It is actually not the default anymore.

Reply:
Ok, then the variable should be `kpi_list` then, no?